src/o365/graph/commands/teams/teams-app-uninstall.ts

```diff
@@ -0,0 +1,144 @@
+import * as request from 'request-promise-native';
+import auth from '../../GraphAuth';
+import Utils from '../../../../Utils';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption, CommandValidate } from '../../../../Command';
+import GraphCommand from '../../GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  appId: string;
+  teamId: string;
+  confirm?: boolean;
+}
+
+class GraphTeamsAppUninstallCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.TEAMS_APP_UNINSTALL}`;
+  }
+
+  public get description(): string {
+    return 'Uninstalls an app from a Microsoft Team team';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+
+    const uninstallApp: () => void = (): void => {
+      auth.ensureAccessToken(auth.service.resource, cmd, this.debug)
+        .then((): request.RequestPromise => {
+          const requestOptions: any = {
+            url: `${auth.service.resource}/v1.0/teams/${args.options.teamId}/installedApps/${args.options.appId}`,
+            headers: Utils.getRequestHeaders({
+              authorization: `Bearer ${auth.service.accessToken}`,
+              accept: 'application/json;odata.metadata=none'
+            })
+          };
+
+          if (this.debug) {
+            cmd.log('Executing web request...');
+            cmd.log(requestOptions);
+            cmd.log('');
+          }
+          return request.delete(requestOptions);
+        })
+        .then((): void => {
+          if (this.verbose) {
+            cmd.log(vorpal.chalk.green('DONE'));
+          }
+          cb();
+        }, (res: Error): void => {
+          this.handleRejectedODataJsonPromise(res, cmd, cb);
+        });
+    };
+    
+    if (args.options.confirm) {
+      uninstallApp();
+    }
+    else {
+      cmd.prompt({
+        type: 'confirm',
+        name: 'continue',
+        default: false,
+            url: `${auth.service.resource}/v1.0/teams/${args.options.teamId}/installedApps/${args.options.appId}`,
```

Comment:
There is no need to pass the URL to the confirmation prompt

Reply:
Thanks! Do you want me to fix this and push another commit to the branch?

Reply:
No need to, I merged the PR already and fixed it when merging the PR.

Reply:
Great, thanks!